server/graphql/v2/mutation/ActivitySubscriptionsMutations.ts

```diff
@@ -0,0 +1,43 @@
+import express from 'express';
+import { GraphQLBoolean, GraphQLNonNull } from 'graphql';
+
+import Channels from '../../../constants/channels';
+import models from '../../../models';
+import { checkRemoteUserCanUseAccount } from '../../common/scope-check';
+import { ActivityAndClassesType } from '../enum/ActivityType';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { ActivitySubscription } from '../object/ActivitySubscription';
+
+const notificationMutations = {
+  setEmailNotification: {
+    type: ActivitySubscription,
+    description: 'Set email notification subscription for requesting logged-in user',
+    args: {
+      type: { type: new GraphQLNonNull(ActivityAndClassesType) },
+      account: {
+        type: AccountReferenceInput,
+        description: 'Scope account which this notification preference is applied to',
+      },
+      active: { type: new GraphQLNonNull(GraphQLBoolean) },
+    },
+    async resolve(_: void, args, req: express.Request) {
+      checkRemoteUserCanUseAccount(req);
+
+      const collective =
+        args.account && (await fetchAccountWithReference(args.account, { loaders: req.loaders, throwIfMissing: true }));
+
+      const where = { UserId: req.remoteUser.id, type: args.type };
+      if (collective) {
+        where['CollectiveId'] = collective.id;
+      }
```

Comment:
I don't get how/where this `where` is being used?

Reply:
Nice catch, it was legacy from the toggle implementation.